common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -654,7 +654,10 @@ def convert(content, input_format, output_format):
         if input_format == 'srt':
 
             if output_format == 'txt':
-                text = SubRipFile.from_string(content.decode('utf-8')).text
+                if six.PY2:
```

Comment:
Changes here are gonna be made by a more comprehensive PR for transcript related issues so you can remove this fix from your PR.